code/modules/cargo/packs/armor.dm

```diff
@@ -0,0 +1,182 @@
+	cost = 1500
+	contains = list(/obj/item/clothing/suit/armor/vest/marine,
+					/obj/item/clothing/head/helmet/bulletproof/x11)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/plasma
+
+/datum/supply_pack/armor/medium_marine_armor
+	name = "Medium Tactical Armor Crate"
+	desc = "One set of well-rounded medium tactical body armor. Plates are attached to the vest and cover the limbs. The set includes a helmet and chestpiece."
+	cost = 3000
+	contains = list(/obj/item/clothing/suit/armor/vest/marine/medium,
+					/obj/item/clothing/head/helmet/bulletproof/x11)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/plasma
+
+//clip
+
+/datum/supply_pack/armor/clip_armor
+	name = "X-11 Armor Crate"
+	desc = "One set of Lanchester Manufacturing-manufactured X-11 armor, sold at a discount to the Confederated League due to a recent contract."
+	cost = 1250
+	contains = list(/obj/item/clothing/suit/armor/vest/bulletproof,
+					/obj/item/clothing/head/helmet/bulletproof/x11/clip,
+					/obj/item/clothing/mask/gas/clip)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/gear
+	faction = /datum/faction/clip
+	faction_discount = 0
+	faction_locked = TRUE
+
+/datum/supply_pack/armor/clip_riot
+	name = "CLIP Riot Armor Crate"
+	desc = "One set of Clover Corp-manufactured Riot armor, traditionally issued to BARD for xenofauna removal, but available to League vessels deployed in the Frontier due to high Xenofauna density."
+	cost = 1000
+	contains = list(/obj/item/clothing/suit/armor/riot/clip,
+					/obj/item/clothing/head/helmet/riot/clip,
+					/obj/item/clothing/mask/gas/clip)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/gear
+	faction = /datum/faction/clip
+	faction_discount = 0
+	faction_locked = TRUE
+
+//gezena
+/datum/supply_pack/armor/pgf_armor
+	name = "Raksha-plate Crate"
+	desc = "One set of standard issue Raksha-plate armor, highly protective and very mobile. The helmet is adjustable for horns."
+	cost = 1500
+	contains = list(/obj/item/clothing/suit/armor/gezena/marine,
+					/obj/item/clothing/head/helmet/gezena)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/gear
+	faction = /datum/faction/pgf
+	faction_discount = 0
+	faction_locked = TRUE
+
+/datum/supply_pack/armor/pgf_armor
+	name = "Navywear Coat Crate"
+	desc = "One replacement navywear coat. The pricing is unfortunately high to discourage distribution to marines, and encourage servicemen to keep track of their jacket."
+	cost = 1000
+	contains = list(/obj/item/clothing/suit/armor/gezena)
+	crate_name = "armor crate"
+	crate_type = /obj/structure/closet/crate/secure/gear
+	faction = /datum/faction/pgf
+	faction_discount = 0
+	faction_locked = TRUE
+
+//jardliner
+
+/datum/supply_pack/armor/hardliner_armor
+	name = "Hardliner Crate"
```

Comment:
Filled with your very own Hardliners
(should probably say _Hardliner **Armor** Crate_)